repos/spack_repo/builtin/packages/qt/qt515_masm_python.patch

```diff
@@ -16,7 +16,7 @@ index b67ee79..b757ee5 100644
  retgen.input = retgen.script
  retgen.CONFIG += no_link
 -retgen.commands = $$QMAKE_PYTHON $$retgen.script > ${QMAKE_FILE_OUT}
-+retgen.commands = "\"$$QMAKE_PYTHON\"" $$retgen.script > ${QMAKE_FILE_OUT}
++retgen.commands = "\"$$QMAKE_PYTHON\"" "\"$$retgen.script\"" > "\"${QMAKE_FILE_OUT}\""
```

Comment:
(question) I guess it would be difficult to come up with a function/filter that handles this and `icu4c` because it looks like these args need an extra layer of wrapping when assigned as a variable.

Reply:
Maybe the logic could be genericized? It would be an incredibly painstaking process however, quoting has long been a thorn in Windows development and this situation is somewhat difficult to identify programmatically (or even as developers, as evidenced by the number of changes I've had to make here)